definitions/contested/json/CaseField/CaseField.json

```diff
@@ -1274,6 +1274,39 @@
     "FieldType": "Text",
     "SecurityClassification": "Public"
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "FinancialRemedyContested",
+    "ID": "additionalHearingDocumentsOptionLabel",
+    "Label": "#### Do you want to upload any other documents ?",
+    "FieldType": "Label",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "FinancialRemedyContested",
+    "ID": "additionalHearingDocumentsOption",
+    "Label": " ",
+    "FieldType": "YesOrNo",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "FinancialRemedyContested",
+    "ID": "additionalListOfHearingDocumentsLabel",
+    "Label": "#### Please upload any additional documents related to hearing.",
+    "FieldType": "Label",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "FinancialRemedyContested",
+    "ID": "additionalListOfHearingDocuments",
+    "Label": " ",
```

Comment:
Could we not have used the same component's label attribute rather than adding another field just for the label?

Reply:
just to make consistance - as it was used before label field - which makes bold.